yaksh/views.py

```diff
@@ -1007,11 +1012,15 @@ def monitor(request, quiz_id=None):
                     attempt_number=last_attempt['last_attempt_num']
                 )
             )
+    csv_fields = ['name', 'username', 'roll_number', 'institute',
+            'department', 'questions', 'total', 'out_of', 'percentage', 'status']
```

Comment:
Can we store the CSV headings ```csv_fields``` in a global variable in ```models.py``` similar to how we store ```question_types``` and ```languages```, this gives it a central place to change instead of hiding it in a view function. Please suggest or feel free to ignore.

Reply:
I too thought of giving it a central place. But models is not the place because in models ```question_types``` and ```languages``` are used by our models, but csv_fields have no relation with models for now so placing there will not be proper.
For now keeping a variable CSV_FIELDS in views itself as the central place.